code/src/App.js

```diff
@@ -1,9 +1,37 @@
-import React from 'react'
+import React, {useState, useEffect} from 'react'
+import {PostedThoughts} from './Thoughts'
+import {MessageInput} from './MessageInput'
 
 export const App = () => {
+const [ thoughts, setThoughts ] = useState([])
+const MESSAGE_URL = 'https://technigo-thoughts.herokuapp.com/'
+
+  useEffect(() => {
+    fetch(MESSAGE_URL)
+      .then((res) => res.json())
+      .then(json => {
+        setThoughts(json)
+      }
+      );
+}, []);
+
+  const onLiked = thoughtId => {
+  const updatedThoughts = thoughts.map(thought => {
+    if (thought._id === thoughtId) {
+      thought.hearts += 1
+    }
+    return thought
+  })
+  
+  setThoughts(updatedThoughts)}
+
   return (
-    <div>
-      Find me in src/app.js!
+    <div className="thoughts">
+      <MessageInput />{
+      thoughts.map(thought => (
+         <PostedThoughts key={thought._id} onLiked={onLiked} thought={thought} />
+     )) 
+     }
```

Comment:
Would nice to have some more work on the indention here so it becomes more easy to read!